as-developers/frontend/README.md

```diff
@@ -0,0 +1,18 @@
+# Django Frontend Framework
+
+## Requirements
+
+- python3
+	$ sudo apt install python3
+- virtualenv
+	$ sudo apt install python3-virtualenv
+	$ virtualenv venv
+	$ chmod 777 venv/bin/activate
+- django
+	$ pip install python
+
+## Run
+
+Enter virtualenv
+	$ . venv/bin/activate
```

Comment:
Hmmm, I'm having trouble activating the virtual environment. I don't have a lot of experience with it, but shouldn't there be some `venv/activate` script someplace?

Reply:
Ah, there is a setup command that I left out. I'll add this in a revision.

Reply:
I've updated the instructions, let me know if you have any trouble.